iscript/src/iscript/macvpn.py

```diff
@@ -44,7 +45,7 @@ async def _create_notarization_zipfile(work_dir, source, dest):
 
     """
     await run_command(
-        ["ditto", "-c", "-k", "--sequesterRsrc", "--keepParent", source, dest],
+        ["ditto", "-c", "-k", "--sequesterRsrc", source, dest],
```

Comment:
The doc string says "keeps parent directory in zip" but I guess that's no longer true? Please update.

Reply:
AIUI this is necessary for gecko because of the multiple-apps-single-zip method. Each source (work/0, work/1, work/2, etc) goes into it's own folder inside the zip.
I was trying to remove as many customized things as possible to make it work.
I'll revert this one to make it the same as gecko.

Reply:
I wasn't commenting on whether this is needed or not.. I'm just saying that the docstring (which is outside of this diff, you need to expand context), says "Keeps parent directory in zip".

The fact that you are removing this flag I think means that it is no longer true, so the docstring should be updated :)